notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        # Pin this version to match that the Dataproc cluster 2.0.0 is using
</pre><p>Small grammar nit: "Pin the Spark version to match a Dataproc 2.0 cluster."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>